users/views.py

```diff
@@ -10,3 +12,22 @@ class SignUpView(CreateView):
     form_class = SignUpForm
     template_name = 'users/signup.html'
     success_url = '/'
+
+
+class FavedSeriesView(LoginRequiredMixin, ListView):
```

Comment:
Shouldn't we place this view (and the associated route) in the `series` app?

It seems more sensible to have the URL be something like `tvshow/followed` than `accounts/faved`, because the resource here is the user's TV show — not their account.